snapshots/cardano-1.26.1.yaml

```diff
@@ -0,0 +1,119 @@
+name: cardano-1.26.1
+
+resolver: lts-17.6
```

Comment:
Could have tried bumping to `17.9`, but didn't do that

Reply:
I bumped it :-)